includes/definitions/discovery/intracom-4wcx.yaml

```diff
@@ -0,0 +1,558 @@
+---
```

Comment:
do you actually need 2 separate OS for 2wcx and 4wcx? seems to share mibs, then it probably should be 1 OS.